src/App.tsx

```diff
@@ -48,47 +54,59 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = React.useState(false);
+  const [sortType, setSortType] = React.useState(SortType.NONE);
+  const goods = getReorderedGoods(goodsFromServer, { sortType, isReversed });
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={`button is-info ${sortType !== SortType.ALPHABET ? 'is-light' : ''}`}
+          onClick={() => {
+            setSortType(SortType.ALPHABET);
+          }}
```

Comment:
Usually, it's better to move such anonymous functions to separate functions.
```suggestion
          onClick={sortByAlphabet}
```